package.json

```diff
@@ -17,6 +17,7 @@
   },
   "devDependencies": {
     "eslint": "^1.0.0",
+    "lodash.assign": "^4.2.0",
```

Comment:
Got it - added `lodash` to `dependencies` and removed `lodash.assign` from `devDependencies`.